travel-journal/index.md

```diff
@@ -0,0 +1,926 @@
+          <div class="content">
+                <p><b>City</b>: {{ journal.city }}</p>
+                <p><b>Country</b>: {{ journal.country }}</p>
+          </div>
+          <div class="content"><img :src="getImageUrl(journal.photo)" alt="Journal Photo" class="journal-photo"/></div>
+          <div class="journal-actions">
+            <button @click="editJournal(journal)" class="edit-button">Edit</button>
+            <button @click="deleteJournal(journal.id)" class="delete-button">Delete</button>
+          </div>
+        </div>
+      </div>
+    </div>
+</template>
+```
+
+And then the following styles:
+
+```css
+<style scoped>
+.read-journal-container {
+  display: flex;
+  flex-direction: column;
+}
+  
+.journal-list {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 20px; 
+  padding: 20px; 
+}
+
+@media (max-width: 1200px) {
+  .journal-list {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+@media (max-width: 800px) {
+  .journal-list {
+    grid-template-columns: repeat(2, 1fr); 
+  }
+}
+
+@media (max-width: 500px) {
+  .journal-list {
+    grid-template-columns: 1fr; 
+  }
+}
+
+.card {
+  border: 1px solid #ddd;
+  border-radius: 4px;
+  padding: 15px;
+  box-shadow: 0 2px 4px rgba(0,0,0,0.1);
+}
+
+.journal-photo {
+  width: 100px;
+  height: 100px;
+  display: block;
+  margin-bottom: 10px;
+}
+
+.journal-actions {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 10px;
+}
+  
+.edit-button, .delete-button {
+  border: none;
+  padding: 10px 15px;
+  border-radius: 4px;
+  cursor: pointer;
+  color: white;
+  font-weight: bold;
+}
+
+.edit-button {
+  background-color: #4CAF50;
+}
+  
+.delete-button {
+  background-color: #f44336;
+}
+  
+.edit-button:hover {
+  background-color: #45a049;
+}
+  
+.delete-button:hover {
+  background-color: #d32f2f;
+}
+
+.content {
+  margin-bottom: 10px;
+  border: 2px solid black;
+  padding: 10px;
+}
+
+.journal-entry {
+  width: 100%; 
+  margin-bottom: 30px;
+}
+</style>
+```
+
+Head over to your Vue.js application over the browser and go ahead to create a journal:
+
+![Creating a journal](./images/9.png)
+
+Once the journal is created successfully, you will be directed to the `Readjournal` view:
+
+![Viewing journal entry](./images/10.png)
+
+Head to the `Journals` collection in your directus project to view your journal entry:
+
+To view the image attached to a journal entry, select the `+` icon at the top right of the `Journals` collection page, select photos and then thumbnails:
+
+![Viewing journal image](./images/11.png)
+
+Onced clicked, you should be able to see the photo related to a journal entry:
+
+![Viewing journal image](./images/12.png)
+
+You can also do this for other columns like description like so:
+
+![Viewing journal description](./images/13.png)
+
+Head to files to see images that was uploaded from your Vue.js application:
+
+![Viewing files in directus](./images/14.png)
```

Comment:
I would remove this - it's kind of unrelated to the application you're building, and more about data exploration inside of the Data Studio. Given the complexity and length of this post, I'd remove all-but-the-most-required teaching.

Reply:
I have renamed the images.